src/electronic-ids/pcsc/FinEID.cpp

```diff
@@ -99,9 +98,9 @@ ElectronicID::PinRetriesRemainingAndMax FinEIDv3::signingPinRetriesLeftImpl() co
 }
 
 byte_vector FinEIDv3::sign(const HashAlgorithm hashAlgo, const byte_vector& hash,
-                           const byte_vector& pin, byte_vector::value_type pinReference,
-                           PinMinMaxLength pinMinMaxLength, byte_vector::value_type keyReference,
-                           byte_vector::value_type signatureAlgo, byte_vector::value_type LE) const
+                           const byte_vector& pin, value_type pinReference,
```

Comment:
To my eyes `value_type` is ambiguous - when I look at this code, it makes me ask "what value_type?". Let's use `byte_type`?

Reply:
I just created alias to shorten code. byte_type should then be declared in libpcsc-cpp